lib/components/Information/Avatar/index.stories.tsx

```diff
@@ -10,13 +10,14 @@ const meta = {
   tags: ["autodocs"],
   args: {
     size: "medium",
+    src: "https://github.com/dani-moreno.png",
```

Comment:
This is not for now, but I think we should create an `ImageSrcProvider` the same way we have a `LinkProvider`. Ideally, the image provider would be able to set the image dimensions as params and even leverage [srcset](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images) for responsive images.